idepix-s2msi/src/main/java/org/esa/snap/idepix/s2msi/operators/cloudshadow/S2IdepixCloudShadowOp.java

```diff
@@ -163,6 +179,37 @@ public void initialize() throws OperatorException {
         setTargetProduct(prepareTargetProduct(sourceResolution, postProduct));
     }
 
+    private float getGeometryMean(Product classificationProduct, String rdnName) {
```

Comment:
Mean is still used in methods and variables. Maybe rename to median?

Reply:
Median will not be correct in most cases, neither. The best thing would be to just call it sun_zenith, geometry, view_azimuth, ...